optimade/server/mappers/entries.py

```diff
@@ -38,6 +39,16 @@ def all_aliases(cls) -> Tuple[Tuple[str, str]]:
             + cls.ALIASES
         )
 
+    @classmethod
+    def all_length_aliases(cls) -> Tuple[Tuple[str, str]]:
+        return (cls.LENGTH_ALIASES,)
+
```

Comment:
Since you already state `LENGTH_ALIASES` is a tuple of tuples, there is no reason to have this method than wraps in into yet another tuple - unless you intend to add to the variable from other sources.
```suggestion
```

Reply:
That is the intention, yeah. I need to think about how this will interact with `provider_fields` and have a play around with it.